setup.py

```diff
@@ -1,9 +1,20 @@
 from setuptools import setup, find_packages
 
+# We define the version number in restructuredtext_lint/__init__.py
+# Here we can't use "import restructuredtext_lint" to access it
+# as "restructuredtext_lint.__version__" as that would tell us the
+# previously installed version (if any).
```

Comment:
It was deliberately verbose - I wanted future readers to be aware while an import would be simpler, it is also wrong.